ext/CUDAExt/cudnn/cudnn.jl

```diff
@@ -0,0 +1,67 @@
+if isdefined(Base, :get_extension)
```

Comment:
Unfortunately no, and I found this out the hard way because I kept missing imports and making the import system freak out. This syntax is documented in https://pkgdocs.julialang.org/dev/creating-packages/#Requires.jl.